docs/help/custom-domains.md

```diff
@@ -1,6 +1,6 @@
 # Custom domains
 
-Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will stay within your brand experience. For this, you'll use a CNAME.
+Your docs are hosted on Bump.sh. However you can set a custom domain: visitors of your docs will then stay within your own name. For this, you'll use a CNAME.
```

Comment:
What bothers you here? I don't feel like it is much clear, however maybe we could just be pragmatic and explain just the how?

```suggestion
While [Bump.sh](http://bump.sh/) hosts your docs using a bump.sh domain, you can also set a custom domain quickly by setting a given CNAME.
```